functions/menu.bash

```diff
@@ -199,6 +199,8 @@ show_main_menu() {
     "42 | Upgrade to openHAB 3"   "Upgrade OS environment to openHAB 3 release" \
     "   | Downgrade to openHAB 2" "Downgrade OS environment from openHAB 3 back to openHAB 2 (DANGEROUS)" \
     "43 | Remote Console"         "Bind the openHAB SSH console to all external interfaces" \
+    "50 | openhab-js"             "JS Scripting: Upgrade to the latest version of the openHAB JavaScript library" \
+    "51 | openhab_rules_tools"    "JS Scripting: Install the latest version of openhab_rules_tools" \
```

Comment:
If we are installing by default, I don't think we need menu option 51 any longer

Reply:
Makes sense, thanks.